Makefile

```diff
@@ -105,8 +105,8 @@ build-tests:
 	cd ./tests && docker build --tag=lisk/service_tests ./
 
 build-local:
-	npm ci
-	cd ./framework && npm ci
+	npm i --registry https://npm.lisk.com
+	cd ./framework && npm i --registry https://npm.lisk.com
```

Comment:
Could using `npm install` create an issue for cases when our code becomes incompatible with new libraries?

Reply:
This change is temporary to bypass current dependency installation issues. This won't make it to the production.